components/automatic-points-table/commons/automatic-points-table.lua

```diff
@@ -9,6 +9,7 @@
 local Arguments = require('Module:Arguments')
 local Class = require('Module:Class')
 local Condition = require('Module:Condition')
+local Custom = require('Module:AutomaticPointsTable/Custom')

```

Comment:
This module doesn't seem to exist, which would throw error.
A simpler approach if this is the only customization would be just taking in an extra `args` (eg. `args.resolveRedirect`) from the template. Then wikis just need to add a param to the template.

Reply:
> This module doesn't seem to exist, which would throw error. A simpler approach if this is the only customization would be just taking in an extra `args` (eg. `args.resolveRedirect`) from the template. Then wikis just need to add a param to the template.

Addressed in https://github.com/Liquipedia/Lua-Modules/pull/1392/commits/f37f024eb0ee4a4601fc39919f86329a32cc3174